draft-ehlen-openpgp-nist-bp-comp.md

```diff
@@ -620,9 +642,6 @@ It is given by the following algorithm, which computes the key encryption key `K
     return KEK
 
 
-Note that the values `ecdhKeyShare` defined in {{ecc-kem}} and `mlkemKeyShare` defined in {{mlkem-ops}} already use the relative ciphertext in the derivation.
-The ciphertext and public keys are by design included again in the key combiner to provide a robust security proof.
```

Comment:
We don't need to justify the construction here, because we are simply adopting it from draft-ietf-openpgp-pqc.

Reply:
Looks good to me.